store/postgres/src/connection_pool.rs

```diff
@@ -479,6 +479,21 @@ impl ConnectionPool {
     }
 }
 
+fn brief_error_msg(error: &dyn std::error::Error) -> String {
```

Comment:
Because those logs can show up in public (e.g., epxlorer) and I didn't want that information in public logs. We still know which shard is affected and could then look the IP up in the config.